.github/workflows/on-demand-workflows.yaml

```diff
@@ -84,3 +84,17 @@ jobs:
           version: v3.3.4
       - run: npm ci
       - run: npm run test:integration
+
+  build-artifacts:
+    if: inputs.run-units == true
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Use Node.js 22
+        uses: actions/setup-node@cdca7365b2dadb8aad0a33bc7601856ffabcc48e # v4.3.0
+        with:
+          node-version: 22
+          cache: "npm"
+      - run: npm ci
+      - run: npm run build
```

Comment:
Nit: `npm run build` is being executed as part of `npm run test:artifacts` so no need to duplicate it here.